test/core.spec.js

```diff
@@ -172,6 +172,45 @@ describe('core specs', () => {
     el.unmount()
   })
 
+  it('Template fragments without bindings must stay in the DOM', () => {
```

Comment:
The use of `<template>` seems confusing to me. Template is a native HTML tag, why does riot override its behavior if it has bindings? 
Can you please answer why it was decided to avoid the <virtual> used in versions 2 and 3?

Reply:
the DOM nodes like `<template>` might be used for Riot.js directives. Without `if, each, is` directives Riot.js will leave your components templates untouched. No side effects will be introduced with the use of `<template>` tags or whathever tag users will use without directives.